### PR TITLE
How to import from Mixpanel Lexicon and Google Sheet

### PR DESCRIPTION
**Pull request by logason**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…racking-plan/importing**](https://docs-qxwicf1we.vercel.app/workspace/tracking-plan/importing) |
|:-:|
|<a href="https://docs-qxwicf1we.vercel.app/workspace/tracking-plan/importing">Screenshot of /workspace/tracking-plan/importing</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/wqsctKQRTJgpTXsIo7FZI93_jsc_.png">(view full size)</a> |







Commit af0641b056d027928df8fd0b9d2ef8ae65c049e2 (https://docs-qxwicf1we.vercel.app).

File: pages/workspace/tracking-plan/importing.mdx [hunk 1]
```diff
@@ -6,49 +6,177 @@ title: 'Importing'
...
+  src={require('../../../images/workspace/importing/importer-select.png')}
+  width="100%"
+  alt="Screenshot of the Avo Importer selecting files"
+/>
+
+4. Review your events and finish the import 🎉
+
+## Importing from Mixpanel
+
+1. Export your Tracking Plan from Mixpanel. Open up Lexicon (in the top right corner click Data Management > Lexicon) and then click the export button:
```
Comment (hugihlynsson) [inline pages/workspace/tracking-plan/importing.mdx:49]: As this section is about exporting from Mixpanel it seems redundant to rephrase it the first step.
```suggestion
1. Open up Lexicon (in the top right corner click Data Management > Lexicon) and then click the export button:
```

**Review by hugihlynsson — APPROVED** [review]
One suggestion otherwise 🔥🔥🔥

File: pages/workspace/tracking-plan/importing.mdx (hunk 1 above)
Comment (logason) [reply pages/workspace/tracking-plan/importing.mdx]: Agreed! shipping

**Pull request merged**